Dataset：https://www.kaggle.com/tanlikesmath/the-oxfordiiit-pet-dataset. 
A little bug : seg-model cannot be loaded normally after save_model()
1. First run the positioning and classification code: positioning_and_classification_model_fit.py
2. Run the segmentation code again: model_fit.py
3. Finally run target_positioning.py in the python console
4、A little bug : model.h5 cannot be loaded normally after save_model()
